src/main/java/net/boomerangplatform/model/TeamWorkflowSummary.java

```diff
@@ -3,12 +3,13 @@
 import java.util.List;
 import org.springframework.beans.BeanUtils;
 import net.boomerangplatform.mongo.entity.FlowTeamEntity;
+import net.boomerangplatform.mongo.model.WorkflowQuotas;
 
 public class TeamWorkflowSummary extends FlowTeamEntity {
 
   private List<WorkflowSummary> workflows;
   
-  private CurrentWorkflowQuotas workflowQuotas;
+  private WorkflowQuotas workflowQuotas;
```

Comment:
change private WorkflowQuotas workflowQuotas;
to private WorkflowQuotas quotas;